moudle.js

```diff
@@ -0,0 +1,94 @@
+const moudle = () => {
+    let posts = []
+    let postId = 0;
+    let commentID = 0;
+    let userName = []
```

Comment:
for something like this you can create an object
`cosnt user = {name: '', id: 0, currentUser: -1}`